torat_client/persit_linux.go

```diff
@@ -23,7 +24,12 @@ var (
 // CheckElevate checks whether the current process has administrator
 // privileges
 func CheckElevate() bool {
-	// TODO: Implement
+	log.Println("[CheckElevate] Reached LINUX root check")
```

Comment:
Yeah, you are def right. I did this because it was so hard for me to track where the code was branching. Since i didn't write this it's a little harder for me to keep track of everything.